Gemfile

```diff
@@ -33,16 +33,16 @@ gem 'huffman', :git => "https://github.com/fgosew/huffman"
 gem 'ruby-graphviz'
 gem 'rsa'
 gem 'gpgmeh'
-gem 'digest-sha3', :git => "http://github.com/izetex/digest-sha3-ruby"
+gem 'sha3'
 gem 'packetfu'
 gem 'net-ntp'
 gem 'CFPropertyList'
 gem 'artii'
+gem 'sqlite3'
```

Comment:
please remove this from the PR until we have some code in SecGen that actually uses this

Reply:
My apologies, I thought I got rid of that before pushing the PR through. I'll remove this.